sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_device.cpp

```diff
@@ -291,15 +291,14 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(
 
     uint32_t ZeSubDeviceCount = Device->SubDevices.size();
     if (ZeSubDeviceCount < 2) {
-      return ReturnValue((ur_device_partition_property_t)0);
+      return ReturnValue((ur_device_partition_t)0);
     }
     bool PartitionedByCSlice = Device->SubDevices[0]->isCCS();
 
     auto ReturnHelper = [&](auto... Partitions) {
       struct {
         ur_device_partition_property_t Arr[sizeof...(Partitions) + 1];
```

Comment:
The return type for `UR_DEVICE_INFO_SUPPORTED_PARTITIONS` is [`ur_device_partition_t`](https://oneapi-src.github.io/unified-runtime/core/api.html#ur__api_8h_1abc90528eef0db5953f07c99722520fdf) so I believe changing to this type should hopefuly fix the issue although I don't have a Windows build locally to verify.

Reply:
Thanks a lot @kbenzie, it helped!